nautobot/extras/models/models.py

```diff
@@ -682,3 +687,49 @@ def log(
             else:
                 log_level = logging.INFO
             logger.log(log_level, str(message))
+
+
+@extras_features("graphql")
+class GraphQLQuery(BaseModel, ChangeLoggedModel):
+    name = models.CharField(max_length=100, unique=True)
+    slug = models.CharField(max_length=100, unique=True)
+    query = models.TextField()
+    variables = models.JSONField(encoder=DjangoJSONEncoder, default=dict, blank=True)
+
+    class Meta:
+        ordering = ("slug",)
+
+    def get_absolute_url(self):
+        return reverse("extras:graphqlquery", kwargs={"slug": self.slug})
+
```

Comment:
Suggest adding a `__str__` method, probably just:

```python
def __str__(self):
    return self.name
```

as this will improve the UI representation of these objects, such as:

![image](https://user-images.githubusercontent.com/5603551/119660743-93dc5c80-bdfd-11eb-81f7-a6edca0d9e8f.png)